compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/DecodeV1.hs

```diff
@@ -534,6 +496,48 @@ decodeBuiltinFunction = \case
   LF1.BuiltinFunctionEQUAL_DECIMAL -> unsupportedDecimal
   LF1.BuiltinFunctionTEXT_TO_DECIMAL -> unsupportedDecimal
 
+  LF1.BuiltinFunctionEQUAL_INT64 -> unsupportedEqual
```

Comment:
Wouldn't this be a problem when reading DARs built with older versions of the compiler?

Reply:
Oh actually you’re right. Somehow I thought LF 1.8 already had those but it doesn’t. Turning it into a draft for now.

Reply:
ah! I was expecting a "that's taken care of elsewhere", glad I mentioned it. Please let me know how you work around it :)

Reply:
luckily the tests I was expecting to fail due to this did also fail. We technically only need to decode types for old LF versions. Maybe that works. But will probably prioritize doing some other stuff first and come back to this later.